opossum-tool/OpossumUI
# feat: offer package version suggestions

### Summary of changes

- do not retry errored queries
- load package versions from deps.dev given a name and system and convert them to autocomplete suggestions
- sort versions by semantic versioning
- display defaults also in attribution view
- add possibility to add suffixes to package attribute labels

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/a5968b6b-bd3b-4081-9f35-9aa61c335aa8)

### Context and reason for change

Closes #2395 

### How can the changes be tested

Add a new attribution for "react" and "npm" and notice that you'll be offered versions via the version autocomplete.
